code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -25,15 +25,27 @@
 	h_style = "blue IPC screen"
 	..(new_loc, "Machine")
 
+/mob/living/carbon/human/monkey
+	density = 0
+
 /mob/living/carbon/human/monkey/New(var/new_loc)
 	..(new_loc, "Monkey")
 
+/mob/living/carbon/human/farwa
+	density = 0
+
 /mob/living/carbon/human/farwa/New(var/new_loc)
 	..(new_loc, "Farwa")
 
+/mob/living/carbon/human/neaera
+	density = 0
```

Comment:
Bad idea. Consider that monkeys are playable in certain situations, enabling them to phase through people without trouble will make certain situations stupid. Case-in-point: ling-monkeys.

Please don't change the density of carbon based mobs.